src/test/lesson_1/Registry.t.sol

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.12;
+
+import "../../lesson_1/Registry.sol";
+import {console} from "forge-std/console.sol";
+import {Vm} from "forge-std/Vm.sol";
+
+abstract contract TestCore {
+    Registry public registry;
+    Vm public vm = Vm(address(uint160(uint256(keccak256("hevm cheat code")))));
+
+    address otherSender;
+
+    event Registered(address to, string name);
+    event Released(address from, string name);
+
+    function setUp() public virtual {
+        registry = new Registry();
+        otherSender = 0x0000000000000000000000000000000000000001;
```

Comment:
if you use the `label('otherSender')` cheat command, the stack traces will show the string and are easier to follow

Reply:
`vm.label(otherSender, 'otherSender')` should be the exact usage.